--- tumbler-0.1.31/acinclude.m4.orig	2015-02-28 16:23:08.000000000 +0100
+++ tumbler-0.1.31/acinclude.m4	2016-12-12 05:33:04.001032871 +0100
@@ -233,7 +233,7 @@
   PKG_CHECK_MODULES([GDK_PIXBUF], [gdk-pixbuf-2.0 >= 2.14],
   [
     dnl Check for libopenraw
-    PKG_CHECK_MODULES([LIBOPENRAW_GNOME], [libopenraw-gnome-1.0 >= 0.0.4], [], [ac_tumbler_raw_thumbnailer=no])
+    PKG_CHECK_MODULES([LIBOPENRAW_GNOME], [libopenraw-gnome-0.1 >= 0.0.4], [], [ac_tumbler_raw_thumbnailer=no])
   ], [ac_tumbler_raw_thumbnailer=no])
 fi
 
